basic/boundCheck5_rand.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2016, 2017 National University of Singapore
```

Comment:
@rasoolmaghareh Should be just 2017 as this is the release date of this program.